torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -58,15 +62,41 @@ C10_DEFINE_int64(
 namespace torch {
 namespace jit {
 
+namespace {
+void removeProfileNodes(Block* b) {
```

Comment:
This exists [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/insert_guards.cpp#L8) (maybe there's a better location for it.)